Objects/enumobject.c

```diff
@@ -83,6 +83,18 @@ enum_new_impl(PyTypeObject *type, PyObject *iterable, PyObject *start)
     return (PyObject *)en;
 }
 
+static int check_keyword(PyObject *kwnames, int index,
+                         const char *name)
+{
+    PyObject *kw = PyTuple_GET_ITEM(kwnames, index);
+    if (!_PyUnicode_EqualToASCIIString(kw, name)) {
+        PyErr_Format(PyExc_TypeError,
+            "'%S' is an invalid keyword argument for enumerate()", kw);
```

Comment:
@JelleZijlstra 
For reducing confusion

```suggestion
            "'%s' is an expected keyword argument for enumerate() but '%S' is given", name, kw);
```

Reply:
I'm not sure that's less confusing, Python in general doesn't say what the expected keyword argument is. I'd rather keep the error message the same as it used to be.